### PR TITLE
Add possibility to get access-tokens from Shuri

### PR DESCRIPTION
**Pull request by oleg1std**
Add possibility to get access-tokens from Shuri
POST https://tr-dev.happytravel.com/token/matsumoto
POST https://tr-dev.happytravel.com/token/shuri

### CODE REVIEW

File: Token/Services/PageFactory.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public PageFactory(IOptions<BaseUrlOptions> options)
 
         public async Task Init()
         {
-            await new BrowserFetcher().DownloadAsync(BrowserFetcher.DefaultRevision);
+            //await new BrowserFetcher().DownloadAsync(BrowserFetcher.DefaultRevision);
```
Comment (kirillta) [inline Token/Services/PageFactory.cs:20]: Why?

Comment (oleg1std) [reply Token/Services/PageFactory.cs:20]: I didn't see the point. Just downloads revision without any usage.

Comment (kirillta) [reply Token/Services/PageFactory.cs:20]: As I understand this line downloads an instance of Chrome, no usage needed

Comment (mingazhev) [reply Token/Services/PageFactory.cs:20]: Anyway, need to check if all works without this line

Comment (mingazhev) [reply Token/Services/PageFactory.cs:20]: This may be a vital hack :)

Comment (kirillta) [reply Token/Services/PageFactory.cs:20]: That may be crucial for clean installs

**Review by mingazhev — APPROVED** [review]

**Review by kirillta — APPROVED** [review]

**Pull request merged**